source/src_lcao/gint_k_rho.cpp

```diff
@@ -8,6 +8,14 @@
 #include "../module_base/blas_connector.h"
 #include "global_fp.h" // mohan add 2021-01-30
 
+#ifdef _OPENMP
+#include <omp.h>
+#endif
+
+#ifdef __MKL
```

Comment:
Is this macro defined inside this project, or pre-defined in `mkl.h`?

Reply:
in Cmakelists.txt and Makefile.system